content/Arduino Cloud/Cloud Editor/If-you-see-an-error-when-performing-an-Over-the-Air-OTA-upload.md

```diff
@@ -102,17 +95,13 @@ This error indicates that the device's memory partition table is improperly form
 
 This error occurs when the device fails to connect to the firmware download endpoint.
 
-Follow these steps:
-
-1. Ensure that the board has a stable connection to Arduino Cloud.
+To resolve this issue, ensure that the board has a stable connection to Arduino Cloud.
 
 ### SHA256 mismatch - Unable to apply the new firmware. Previous firmware detected after restart {#sha256mismatch}
 
 This error means that the device restarted with a SHA (fw identifier) that is different from the target one and the SHA is equal to the one installed when the OTA process is started. This usually happens when the device restarts during the OTA process.
 
-Follow these steps:
-
-1. Check if the sketch running on the device is unstable and causes frequent restarts.
+To resolve this issue, check if the sketch running on the device is unstable and causing frequent restarts.
```

Comment:
```suggestion
Follow these steps:

1. Ensure the device is connected to a stable and uninterrupted power source.

2. Upload a simple example sketch (such as BareMinimum) to prevent any resets caused by faulty code.

3. If you're unable to program the device, try putting it in bootloader mode.
```

Reply:
[697e941](https://github.com/arduino/help-center-content/pull/457/commits/697e941807a332aaab63eb1e62678905a9ed4e59)